web/application/controllers/Airqo.php

```diff
@@ -57,35 +57,35 @@ public function projects()
 		$this->load->view('lib/footer',$data);
 	}
 
-	public function blog($category = null)
-	{
-		if($category == null) {
-			$data['title'] = 'Blog';
-			$this->logVisitedPage($data['title']);
-			$data['posts'] = $this->AirqoModel->get_post();
-			$data['suggestions'] = $this->AirqoModel->get_home_random();
-			$data['categories'] = $this->AirqoModel->get_categories();
-
-			$this->load->view('lib/header',$data);
-			$this->load->view('airqo-blog-new',$data);
-			$this->load->view('lib/footer',$data);
-		} else{
-			$data['title'] = 'Blog';
-			$this->logVisitedPage($data['title']);
-			$data['posts'] = $this->AirqoModel->get_post_categories($category);
-			$data['suggestions'] = $this->AirqoModel->get_home_random();
-			$data['categories'] = $this->AirqoModel->get_categories();
-
-			$this->load->view('lib/header',$data);
-			$this->load->view('airqo-blog-new',$data);
-			$this->load->view('lib/footer',$data);
-		}
-	}
+	// public function blog($category = null)

```

Comment:
Could we remove/delete these comments?
Unless we plan to use them in the near future?

Reply:
resolved